### PR TITLE
CI: Add basic csv validation checks

### PR DESCRIPTION
**Pull request by tangkong**
I rolled a quick csv validation check for blank lines and duplicate pvs.  

I just made a quick python script, instead of trying to set up pytest.   I wonder if that was more or less lazy.

Also fixes the main/master discrepancy that stopped the CI from pushing csvs to master before

### CODE REVIEW

File: .github/actions/validate_csv/action.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: "validate csv"
+description: "validate csv data"
+
+runs:
+  using: "composite"
+  steps:
+    - run: echo "Validating CSV data"
+      shell: bash
+    - run: python Scripts/validate_csvs.py Spreadsheet/
```
Comment (klauer) [inline .github/actions/validate_csv/action.yml:9]: A thought: it'd be good to bake this into a pre-commit configuration, and just have CI run pre-commit.

File: Scripts/validate_csvs.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+import click
+import pandas as pd
+
+
+def check_duplicate_pvs(df: pd.DataFrame):
```
Comment (klauer) [inline Scripts/validate_csvs.py:7]: Nit: return type annotations on any of these?

File: dev-requirements.txt
Comment (klauer) [inline dev-requirements.txt:1]: Nit: are these really dev requirements? Somewhere between "regular" ones and dev ones, I suppose...

**Review by klauer — APPROVED** [review]
Looks good overall. Some ideas for (arguable) improvement.

File: .github/actions/validate_csv/action.yml (hunk 1 above)
Comment (tangkong) [reply .github/actions/validate_csv/action.yml:9]: I totally agree, that does seem like a nice place for this

Though ultimately the people making PR's will be scientists, and they seem to have their hands full even forking the repo.

File: dev-requirements.txt
Comment (tangkong) [reply dev-requirements.txt:1]: ci-requirements?  Since this only runs in CI it felt like dev-reqs, but I suppose the scripts are included

**Pull request merged**